### DISCUSSION TITLE
Scatter plot zoom for RTK analysis

### DISCUSSION THREAD

**Discussion by nseidle** [semuconsulting/PyGPSClient#160] [Ideas and Feature Requests]
Hi,

I love PyGPSClient but the scatter plot is limited in its zoom capabilities: 

![image](https://github.com/user-attachments/assets/be04a862-04c0-4fc0-ac95-8491091a7bed)

Above is what an RTK Fix with ~20mm of accuracy looks like over 30 seconds. You can't glean much.

![image](https://github.com/user-attachments/assets/32c063a4-36b4-4360-bcb3-f0120f39482a)

Above is QGNSS (by Quectel). Note the square gradations (not as good as circular plots). But the zoom in/out capabilities are very handy.

![image](https://github.com/user-attachments/assets/98116b0b-eaf3-4681-81b5-fd1eb90cf879)

Above is ucenter with a 0.05m manual override.

Is there any chance we could get better zoom in/out capabilities of the scatter plot? These zoomed in views allow us to quantify and qualify how a given RTK engine performs over a period of time, and from different correction sources. 

Double clicking to clear the plot is also extremely helpful - thanks for that!

(And a special thank you for the links to SparkFun antennas on the RTK page!)

**Comment by semuadmin** [comment]
> (And a special thank you for the links to SparkFun antennas on the RTK page!)

You're welcome - anytime SparkFun feel like sponsoring the project or donating GNSS kit, I might get round to considering their feature requests 😜

**Comment by nseidle** [comment]
By all means! Coffee sent. Let me know a good address to send GNSS gear and I will get it shipped. Do you need something portable like [Facet](https://www.sparkfun.com/products/19984) or [Torch](https://www.sparkfun.com/products/25662), or do you need a [ZED breakout board](https://www.sparkfun.com/products/16481) + [antenna](https://www.sparkfun.com/products/21801) + [cable](https://www.sparkfun.com/products/21740) + [mount](https://www.sparkfun.com/products/21257), etc?

**Reply by semuadmin** [reply]
You are a gentleman and a scholar, sir. My direct email is semuadmin@semuconsulting.com, we can pick things up from there if you like.

Re. #160 and #161 - see https://github.com/semuconsulting/PyGPSClient/discussions/161#discussioncomment-11105064.

**Comment by semuadmin** [comment]
Delivered in version 1.4.27 now available on PyPi
